### PR TITLE
ci: expand battle test repo list

### PR DESCRIPTION
**Pull request by elsapet**
## Description
* Add more repos to KPI list

Relates to #895

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

File: .github/workflows/kpi_scans.yml [hunk 1]
```diff
@@ -14,6 +14,16 @@ jobs:
...
+          - https://github.com/forem/forem
+          - https://github.com/freeCodeCamp/chapter
+          - https://github.com/mastodon/mastodon
+          - https://github.com/frab/frab
+          - https://github.com/discourse/discourse
+          - https://github.com/diaspora/diaspora
+          - https://github.com/TryGhost/Ghost
+          - https://github.com/wekan/wekan
+          - https://gitlab.com/gitlab-org/gitlab
+          - https://github.com/grafana/grafana
```
Comment (elsapet) [inline .github/workflows/kpi_scans.yml]: @gotbadger I feel this list is straight-forward / low effort enough to update, but let me know if we still want a separate JSON file for these repositories

Comment (gotbadger) [reply .github/workflows/kpi_scans.yml]: Yes I would like this loaded from a json file

**Comment by gotbadger** [comment]
proably should put the json file in the `kpi_scan` folder

**Review by gotbadger — APPROVED** [review]
move `kpi_repo_list.json` to the `kpi_scan` folder but otherwise looks fine

**Pull request merged**